### ISSUE TITLE
Getting Started Expected Location Update

### ISSUE THREAD

**Issue by scottschreckengaust** [newcontext-oss/kitchen-terraform#357]
Running the Getting Started does not verify for:

test-kitchen (2.2.5)
kitchen-terraform (5.1.0)
terraform (0.12.6)
ruby (2.5.5p157)

The `main.tf` places the `foobar` file in the working directory.

https://github.com/newcontext-oss/kitchen-terraform/blob/0cfa922967c085525d1f772acc99373a9707f43c/docs/getting_started.html#L200

Should the InSpec contents for the `test/integration/kt_suite/controls/basic.rb` be:
```
# frozen_string_literal: true

control "file_check" do
  describe file('test/fixtures/tf_module/foobar') do
    it { should exist }
  end
end
```

not:
```
# frozen_string_literal: true

control "file_check" do
  describe file('.kitchen/kitchen-terraform/kt-suite-terraform/foobar') do
    it { should exist }
  end
end
```

Is there a simple way to do this introspection of the `.kitchen.yml` file's `root_module_directory` configuration?:
```
  # TODO: access the `root_module_directory` configuration
  #   Kitchen::Terraform::ConfigAttribute::RootModuleDirectory
  #   :@root_module_directory
  my_root_module_directory = 'test/fixtures/tf_module'
  describe file(File.join(my_root_module_directory, 'foobar')) do
    it { should exist }
  end
```

**Comment by aaron-lane** [comment]
Hi @scottschreckengaust. Thank you for your interest in the project!

You're right; the tutorial appears to have fallen out of date. I'll try to address this soon.

**Comment by aaron-lane** [comment]
Fixed by #378.
